components/user-mgt/org.wso2.carbon.identity.user.profile.ui/src/main/java/org/wso2/carbon/identity/user/profile/ui/client/UserProfileUIUtil.java

```diff
@@ -105,4 +106,26 @@ private static boolean isUsernameEncryptionEnabled() throws CarbonException, Use
         return Boolean.parseBoolean(AdminServicesUtil.getUserRealm().getRealmConfiguration()
                 .getRealmProperties().get(ENCRYPT_USERNAME_IN_URL));
     }
+    /**
+     * Check whether the account is attached with account lock bypassable role.
+     *
+     * @param userName user name whos roles needs to be listed
+     * @return true or false based on user roles
+     * @throws CarbonException, UserStoreException
+     */
+    public static boolean isAccountLockable(String userName)
+            throws CarbonException, UserStoreException {
+        boolean isLockable = true;
+        String[] roleList = AdminServicesUtil.getUserRealm().getUserStoreManager().getRoleListOfUser(userName);
+        if (roleList != null) {
+            for (String roleName : roleList) {
+                if (roleName.equals("Internal/bypassaccountlock")) {
```

Comment:
shall we define a constant for "Internal/bypassaccountlock"

Reply:
done